documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1352,6 +1352,16 @@ Note that this setting is not recommended and is planned for removal in a future
 |Whether field names will be sanitized to adhere to Avro naming requirements.
 See xref:configuration/avro.adoc#names[Avro naming] for more details.
 
+|`database.server.timezone`
```

Comment:
@jvenant Is the documentation up to date?

Reply:
it wasn't. But now it is